src/plugins/urls.ts

```diff
@@ -3,17 +3,15 @@ import urlJoin from "url-join";
 
 import { Dialog, Pagination, SingleAction } from "../types";
 
-export const pluginsSection = "/plugins/";
+export const pluginSection = "/plugins/";
 
-export const pluginsListPath = pluginsSection;
-export type PluginsListUrlQueryParams = Pagination & SingleAction;
-export const pluginsListUrl = (params?: PluginsListUrlQueryParams) =>
-  pluginsListPath + "?" + stringifyQs(params);
+export const pluginListPath = pluginSection;
+export type PluginListUrlQueryParams = Pagination & SingleAction;
+export const pluginListUrl = (params?: PluginListUrlQueryParams) =>
+  pluginListPath + "?" + stringifyQs(params);
```

Comment:
A bit hackish on 1st glance. Multiple addition operators I mean. Not sure though if changing to template literals would sustain the readability.

Reply:
Agreed, let's make this an issue